src/_image_resample.h

```diff
@@ -699,8 +702,8 @@ static void get_filter(const resample_params_t &params,
 }
 
 
-template<typename color_type>
-void resample(
+template<typename color_type, bool is_large = false>
```

Comment:
Would this be a better name?
```suggestion
template<typename color_type, bool use_32bit = false>
```

Reply:
Since there's not really been much change to memory usage, I'm going to drop the template change and just always use the larger scan lines. That'll save us some code space, and template complications.